llvm/utils/update_mir_test_checks.py

```diff
@@ -87,13 +87,14 @@ def __call__(self, args, ir):
 
 
 class Run:
-    def __init__(self, prefixes, cmd_args, triple):
+    def __init__(self, prefixes, cmd_args, preprocess_cmd, triple):
```

Comment:
Can you split this into a separate commit?

Reply:
Yeah no problem